core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -106,7 +114,7 @@ public String toString() {
   protected class BaseMetastoreCatalogTableBuilder implements TableBuilder {
     private final TableIdentifier identifier;
     private final Schema schema;
-    private final ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builder();
+    private final Map<String, String> propertiesBuilder = Maps.newHashMap();
```

Comment:
The variable name here is misleading now, but leaving it as is to avoid conflicts. Let me know if we should change this instead.

Reply:
I agree. It's worth the rename to `properties`.